src/utils/utils.ts

```diff
@@ -47,8 +77,8 @@ export const getNumItems = async (uuid: string) => {
 
     if (response.status === 200) {
       const data = await response.json();
-      const numItems = data.nyplAPI.response.numItems;
-      return numItems;
+      const responseData = data.nyplAPI.response;
+      return responseData;
     } else {
       return 0;
```

Comment:
Now that this function doesn't return `numItems`, return an empty object (or perhaps undefined?) instead of a 0.